debian/jitsi-meet-web.install

```diff
@@ -15,5 +15,5 @@ resources/robots.txt	/usr/share/jitsi-meet/
 resources/*.sh			/usr/share/jitsi-meet/scripts/
 pwa-worker.js			/usr/share/jitsi-meet/
 manifest.json			/usr/share/jitsi-meet/
-load-test/*.html		/usr/share/jitsi-meet/load-test/
-load-test/libs  		/usr/share/jitsi-meet/load-test/
+resources/load-test/*.html	/usr/share/jitsi-meet/load-test/
```

Comment:
Side note, I wouldn't ship this on the Debian package.

Reply:
It is not enabled and files are not accessible if not enabled through nginx.